recourse_methods/mrmc_method.py

```diff
@@ -57,26 +57,26 @@ def __init__(self,
         
         Args:
             dataset: The dataset to provide recourse over.
-            preprocessor: The dataset's preprocessor.
+            adapter: The dataset's adapter.
             label_column: The columb providing binary classification outcomes.
             positive_label: The value of a positive classification outcome.
             alpha: The alpha function to use during recourse generation.
             rescale_direction: A function for rescaling the recourse."""
-        self.X = MRM.process_data(dataset, preprocessor, label_column, positive_label)
-        self.preprocessor = preprocessor
+        self.X = MRM.process_data(dataset, adapter, label_column, positive_label)
```

Comment:
Helpful to rename X to a more descriptive name. You can include what type of data this is in the variable name.